src/server/routes/backend-wallet/remove.ts

```diff
@@ -0,0 +1,64 @@
+import { Static, Type } from "@sinclair/typebox";
+import { FastifyInstance } from "fastify";
+import { StatusCodes } from "http-status-codes";
+import { Address } from "thirdweb";
+import { deleteWalletDetails } from "../../../db/wallets/deleteWalletDetails";
+import { createCustomError } from "../../middleware/error";
+import { standardResponseSchema } from "../../schemas/sharedApiSchemas";
+
+const requestSchema = Type.Object({
+  walletAddress: Type.Optional(Type.String()),
```

Comment:
why do we make it optional here, but validate its existence later?

Reply:
Hmm you're right I copied it from a different route. Should just be required.